packages/babel-plugin/src/__tests__/expression-evaluation.test.ts

```diff
@@ -454,5 +454,46 @@ describe('import specifiers', () => {
         'ax(["_19pk19vg"])',
       ]);
     });
+
+    it('statically evaluates a TS const expression', () => {
+      const actual = transform(
+        `
+        import '@compiled/react';
+
+        const styles = { color: 'red' } as const;
+
+        <div css={{ ...styles }} />;
+      `,
+        {
+          parserBabelPlugins: ['typescript', 'jsx'],
+        }
+      );
+
+      expect(actual).toIncludeMultiple(['._syaz5scu{color:red}', 'ax(["_syaz5scu"])']);
+    });
+
+    it('statically evaluates a TS const expression in a resolved binding', () => {
```

Comment:
Hey @JakeLane what's the difference with the test above? Are you just testing with different Compiled APIs?